pyccel/codegen/compiling/compilers.py

```diff
@@ -385,7 +385,7 @@ def compile_shared_library(self, compile_obj, output_folder, verbose = False, sh
         # Collect compile information
         exec_cmd, includes, libs_flags, libdirs_flags, m_code = \
                 self._get_compile_components(compile_obj, accelerators)
-        linker_libdirs_flags = ['-Wl,-rpath' if l == '-L' else l for l in libdirs_flags]
+        linker_libdirs_flags = ['-Wl,-rpath' if l == '-L' and self._info['exec'] != 'nvcc' else l for l in libdirs_flags]
```

Comment:
@EmilyBourne not sure if this hack is the correct way to do this but i just wanted to not use the linking flags when the compiler is `nvcc`

Reply:
Can you explain why you didn't want to use the linking flags?

Reply:
This seems to be related to this:

https://forums.developer.nvidia.com/t/solved-error-while-linking-with-nvcc/46476/6

Reply:
See #13 & #14